packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/CameraTest.java

```diff
@@ -142,7 +142,10 @@ public void before() {
             mockDartMessenger,
             mockCameraProperties,
             resolutionPreset,
-            enableAudio);
+            enableAudio,
+            Integer.valueOf(15),
+            Integer.valueOf(200000),
+            Integer.valueOf(32000));
```

Comment:
All of the changes in this file are just new parameters. Where are the tests that they work?

Reply:
Preparing tests...

Reply:
unit and integration tests are added.